datafusion/common/src/types/native.rs

```diff
@@ -433,4 +433,29 @@ impl NativeType {
             UInt8 | UInt16 | UInt32 | UInt64 | Int8 | Int16 | Int32 | Int64
         )
     }
+
+    #[inline]
```

Comment:
Do we need these compiler hints?

Reply:
I think so. we used it a lot in arrow-rs for trivial function (like matches!()) too.